src/main/java/uk/gov/hmcts/divorce/common/event/SubmitConditionalOrder.java

```diff
@@ -72,7 +72,7 @@ public void configure(ConfigBuilder<CaseData, State, UserRole> configBuilder) {
         log.info("Submit conditional order about to submit callback invoked for case id: {}", details.getId());
 
         CaseData data = details.getData();
-        data.getConditionalOrder().setDateSubmitted(LocalDateTime.now(clock));
+        data.getConditionalOrder().setApplicant1SubmittedDate(LocalDateTime.now(clock));
```

Comment:
if we're going to assume app1 is submitting (which we should) then you need to clean up the permissions for the event: 

```
.grant(CREATE_READ_UPDATE, APPLICANT_1_SOLICITOR, CREATOR, CITIZEN, APPLICANT_2)
```

should be: 

```
.grant(CREATE_READ_UPDATE, APPLICANT_1_SOLICITOR, CREATOR)
```

(app1 or app1 solicitor)

Reply:
Also, we need to consider the situation where app2 has submitted first. So the states probably need to be:

```
.forStates(ConditionalOrderDrafted, ConditionalOrderPending)
```

And we need a check on the return state so that if it's a joint application AND originalState == ConditionalOrderDrafted THEN state = ConditionalOrderPending ELSE AwaitingLegalAdvisorReferral